README.rst

```diff
@@ -71,6 +71,10 @@ Remember to add a ``SECRET_KEY`` setting!
   The manage command will automatically reload the core to ensure schema
   changes take effect.
 
+- Run the migrations
+
+    python manage.py migrate
+
```

Comment:
I guess I assume this is a standard django step to do, but it probably doesn't hurt to document.